modules/core/src/v2/coins/cspr.ts

```diff
@@ -262,13 +269,30 @@ export class Cspr extends BaseCoin {
       const amount = accountLib.Cspr.Utils.getTransferAmount(tx.casperTx.session);
       const toAddress = accountLib.Cspr.Utils.getTransferDestinationAddress(tx._deploy.session);
       const outputs: TransactionOutput[] = [];
+      const operations: TransactionOperation[] = [];
       // TODO(https://bitgoinc.atlassian.net/browse/STLX-677): define outputs for different types of tx
       if (tx.type === accountLib.BaseCoin.TransactionType.Send) {
         outputs.push({
           address: toAddress,
           amount,
           coin: self.getChain(),
         });
+      } else if (tx.type === accountLib.BaseCoin.TransactionType.StakingLock) {
+        const validator = accountLib.Cspr.Utils.getValidatorAddress(tx._deploy.session);
+        operations.push({
+          type: 'StackingLock',
```

Comment:
Typo in `StackingLock`. We should get this from the TransactionType enum to prevent typos.
```suggestion
          type: accountLib.BaseCoin.TransactionType.StakingLock,
```